spec/serializers/spree_signifyd/order_serializer_spec.rb

```diff
@@ -15,8 +15,17 @@ module SolidusSignifyd
         order.payments.last.update(avs_response: "M", cvv_response_code: "M")
       end
 
-      it 'matches the SIGNIFYD V2 api' do
-        expect(serialized_order).to match_schema('v2/case.json')
+      if SolidusSupport.combined_first_and_last_name_in_address?
+
+        it 'matches the SIGNIFYD V2-with-combined-name api' do
```

Comment:
Reading twice, I don't get why this is necessary. We were providing the user's full name and now name, which is the same thing. What's the difference?

Reply:
The schemas that they are matching to have one section which is different due to the versions:

```
     "required": [
        "fullName",
        "confirmationEmail",
        "deliveryAddress"
      ]
 ```

vs.

```
      "required": [
        "name",
        "confirmationEmail",
        "deliveryAddress"
      ]
 ```

Reply:
And what's the difference in their perspective between the two? Asking because for us `fullName` and `name` are actually the same thing. So if we pass our `.name` or `.full_name` they will have the same value, so I don't get why we should change how we call it.

Reply:
Thank you for the clarification, I have updated it to be in line with the API - my apologies for overlooking that!